### PR TITLE
Add backwards compatibility for output template model python scripts

### PR DESCRIPTION
**Pull request by Joeper214**
## Motivation and Context

Due to the latest update of blueoil v0.4.0 currently the python interface of `run.py` and `usb_camera_demo.py` was changed. But we need backward compatibility of the previous command.

## Description

Update `run.py` and `usb_camera_demo.py` commands to accept the `-l` parameter for the command interface. 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature / Optimization (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by Joeper214** [comment]
@ruimashita @iizukak please review if you have time. 🙇

File: output_template/python/usb_camera_demo.py [hunk 1]
```diff
@@ -261,10 +261,14 @@ def run(model, config_file):
...
+    "-l",
     "--model",
     type=click.Path(exists=True),
-    help=u"Inference Model filename",
-    default="../models/lib/lib_fpga.so",
+    help=u"""
+        Inference Model filename
+        (-l is deprecated please use -m instead)
+    """,
+    default="../models/minimal_graph_with_shape.pb",
```
Comment (ruimashita) [inline output_template/python/usb_camera_demo.py]: I think default value is better to use FPGA lib. why do you change default value?

Comment (Joeper214) [reply output_template/python/usb_camera_demo.py]: @ruimashita Oh. I see. There was no reason why I changed it.  This was a mistake. Thanks!

**Comment by Joeper214** [comment]
@ruimashita I updated. Please check again.

**Comment by ruimashita** [comment]
@Joeper214 
Thanks 👍 
Could you prompt deprecated message when user choice `-l` option ?

**Comment by Joeper214** [comment]
@ruimashita Thanks fixed.

> Could you prompt deprecated message when user choice -l option ?

**Comment by ruimashita** [comment]
I verified that
```
$ python ~/blueoil/output_template/python/run.py -l ../models/lib/lib_x86.so  -c ../models/meta.yaml -i ~/blueoil/lmnet/tests/fixtures/sample_images/cat.jpg
Deprecated warning: -l is deprecated please use -m instead
Output: (before post process)
[[[[0. 0. 0. ... 0. 0. 0.]
   [0. 0. 0. ... 0. 0. 0.]

....
```

File: output_template/python/usb_camera_demo.py (hunk 1 above)
Comment (ruimashita) [reply output_template/python/usb_camera_demo.py]: This was not changed :sob::sob::sob:

Comment (ruimashita) [reply output_template/python/usb_camera_demo.py]: @Joeper214 Could you fix this ??

Comment (Joeper214) [reply output_template/python/usb_camera_demo.py]: @ruimashita Sure. I'll do that.

**Pull request merged**